src/main/java/com/celements/course/CourseScriptService.java

```diff
@@ -185,6 +186,18 @@ public long getRegistrationCount(DocumentReference courseDocRef, RegistrationSta
     return retVal;
   }
 
+  public long getRegistrationCount(DocumentReference courseDocRef,
+      List<ParticipantStatus> ignorList) {
```

Comment:
naming:
- "ignore" with e
- the name should state what is ignored, e.g. ignoreStates